### DISCUSSION TITLE
we can maybe use steam to run bookmarkets ?

### DISCUSSION THREAD

**Discussion by SIMONHSCH** [3kh0/ext-remover#1138] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
if you go to [steam](https://store.steampowered.com/about/) you can click on a link to install steam on chromebook.
the link is `chromeos-steam://install` 
If you're a nerd like me you know that only urls that can access to ur browser like [https://chromewebstore.google.com/](https://chromewebstore.google.com/) can put this type of link
if [steam](https://store.steampowered.com/about/) can access to ur browser, it means that you can disable extensions when you use bookmarkets

**Comment by s0urce-c0de** [comment]
Just an idea, but can we use uBO to inject JS into steam?

**Comment by xoidex** [comment, marked as answer]
nuh uh

only the link "chromeos-steam://install" gets access
for example it works on any site that can generate a hyperlink
ps. it doesn't work because of hardware restrictions
